geoserver/1.9/README.md

```diff
@@ -0,0 +1,133 @@
+- Target audience: Teams that need a highly-available, pre-tuned GeoServer deployment.
+- Scope: Install and use GeoServer.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Configure Shared Storage](#configure-shared-storage)
+- [Install GeoServer](#install-geoserver)
+- [Use GeoServer](#use-geoserver)
+- [Extensions](#extensions)
+- [Uninstall GeoServer](#uninstall-geoserver)
+
+## Prerequisites
+
+- A running DC/OS 1.9 cluster with a total of 4.1 CPUs and 2 GB of RAM available within the cluster.
+- Marathon-LB Universe package installed.
+- DNS or access to local machine configuration for host resolution.
+- [DC/OS CLI](https://dcos.io/docs/1.9/usage/cli/install/) installed for ease of access to cluster nodes.
+
+
+## Configure Shared Storage
+
+If your cluster already has some form of storage shared among all private nodes, this step can be skipped. This example
+configuration sets up an NFS server on the master and connects all nodes to the master as NFS clients.
+
+- Connect to the master using the CLI: `dcos node ssh --leader --master-proxy`
+- The following commands must be executed on the master:
+
+    ```bash
+    sudo su -
+    mkdir /shared
+    mkdir -p /var/lib/nfs
+    touch /var/lib/nfs/etab
+    echo '/shared *(rw,sync,no_subtree_check,fsid=0,no_root_squash)' >> /etc/exports
+    chmod 644 /etc/exports
+    chmod 777 /shared
+    iptables -A INPUT -p tcp -m tcp --dport 2049 -j ACCEPT
+    iptables -A INPUT -p tcp -m tcp --dport 111 -j ACCEPT
+    iptables -A INPUT -p udp -m udp --dport 111 -j ACCEPT
+    systemctl restart nfsd.service
+    systemctl restart rpcbind.service
+    systemctl restart rpc-mountd.service
+    ```
+
+- Identify all slave IDs: `dcos node | tail +2 | awk '{print $3}'`
+- Connect to each of the nodes: `dcos node ssh --master-proxy --mesos-id=ONE-OF-ABOVE-LISTED-IDS`
+- The following commands must be executed on each of the nodes:
+
+    ```bash
+    sudo su -
+    mkdir /shared
+    mkdir /var/lib/nfs
+    systemctl enable rpc-statd.service
+    systemctl restart rpc-statd.service
+    mount -t nfs master.mesos:/shared /shared
+    ```
+
+*Note*: The described configuration exports an NFS share on the master from the root volume. If there is going to be any
+appreciable storage, this should be moved to a dedicated volume for this purpose. For the purposes of GeoServer
+configuration storage, this space will be minimal (<10MiBs).
+
+
+## Install GeoServer
+
+To install GeoServer, navigate to Universe from the DC/OS UI and select GeoServer. Advanced Installation should be used
+and your chosen address should be entered in to the `geoserver.virtual-host` configuration field. For this tutorial, we
+will continue to reference the default in example URLs - be sure to substitute the appropriate value for your
+environment.
+
+After this, you should see the 2 `geoserver` services running via the `Services` tab of the DC/OS UI:
+
+![GeoServer DC/OS service](img/services.png)
```

Comment:
Agreed. I added marathon-lb to screenshot to better reflect what you should actually see. The Universe not being in your view was actually the reason you are having issues. https://github.com/mesosphere/universe/pull/1145 added support for DC/OS 1.9, which I was testing this example against my own build of the Universe. You will be able to deploy if you just change the image being used to `appliedis/dcos-geoserver:1.2.0`. Once the mentioned PR is merged, this will be in the official Universe.

<img width="1264" alt="screen shot 2017-05-03 at 10 17 51 pm" src="https://cloud.githubusercontent.com/assets/4768496/25688641/df5f90e6-304e-11e7-98bf-30ba412e0f51.png">